media/static-bundles.json

```diff
@@ -1406,6 +1412,13 @@
         "js/base/error-pages.js"
       ],
       "name": "page_not_found"
+    },
+    {
+      "files": [
+        "js/firefox/installer-help-experiment.js",
+        "js/libs/mozilla-traffic-cop.js"
```

Comment:
Seems this is still throwing an error here, because you added traffic cop after the script that tries to use it.